python/ray/util/spark/utils.py

```diff
@@ -234,6 +270,11 @@ def _calc_mem_per_ray_worker_node(
 RAY_ON_SPARK_WORKER_PHYSICAL_MEMORY_BYTES = "RAY_ON_SPARK_WORKER_PHYSICAL_MEMORY_BYTES"
 RAY_ON_SPARK_WORKER_SHARED_MEMORY_BYTES = "RAY_ON_SPARK_WORKER_SHARED_MEMORY_BYTES"
 
+# User can manually set these environment variables on spark driver node
+# if ray on spark code accessing corresponding information failed.
+RAY_ON_SPARK_DRIVER_PHYSICAL_MEMORY_BYTES = "RAY_ON_SPARK_DRIVER_PHYSICAL_MEMORY_BYTES"
```

Comment:
should it be HEAD or DRIVER?

Reply:
One ray head node should be started on the spark driver node if user want to use this feature. So the `RAY_ON_SPARK_DRIVER_PHYSICAL_MEMORY_BYTES` means the memory size of the whole driver node.